docs/my-website/docs/caching/all_caches.md

```diff
@@ -144,7 +144,62 @@ assert response1.id == response2.id
 
 </TabItem>
 
+<TabItem value="qdrant-sem" label="qdrant-semantic cache">
 
+You can set up your own cloud Qdrant cluster by following this: https://qdrant.tech/documentation/quickstart-cloud/
+
+To set up a Qdrant cluster locally follow: https://qdrant.tech/documentation/quickstart/
+```python
+import litellm
+from litellm import completion
+from litellm.caching import Cache
+
+random_number = random.randint(
+    1, 100000
+)  # add a random number to ensure it's always adding / reading from cache
+
+print("testing semantic caching")
+litellm.cache = Cache(
+    type="qdrant-semantic",
+    qdrant_host_type="cloud", # can be either 'cloud' or 'local'
+    qdrant_url=os.environ["QDRANT_URL"], 
+    qdrant_api_key=os.environ["QDRANT_API_KEY"],
+    qdrant_collection_name="your_collection_name", # any name of your collection
+    similarity_threshold=0.7, # similarity threshold for cache hits, 0 == no similarity, 1 = exact matches, 0.5 == 50% similarity
+    qdrant_quantization_config ="binary", # can be one of 'binary', 'product' or 'scalar' quantizations that is supported by qdrant
+    qdrant_semantic_cache_embedding_model="text-embedding-ada-002", # this model is passed to litellm.embedding(), any litellm.embedding() model is supported here
+)
+
+response1 = completion(
+    model="gpt-3.5-turbo",
+    messages=[
+        {
+            "role": "user",
+            "content": f"write a one sentence poem about: {random_number}",
+        }
+    ],
+    max_tokens=20,
+)
+print(f"response1: {response1}")
+
+random_number = random.randint(1, 100000)
+
+response2 = completion(
+    model="gpt-3.5-turbo",
```

Comment:
can you pass `mock_response=hi` we don't want the caching tests to make a call to the LLM API

Reply:
mock_response is a param you can pass to litellm.completion

Reply:
Apparently when I provide two different mock responses (let's say 'hi1' and 'hi2') to the two acompletion request in the first test, it raises an assertion error as both the requests return 'hi1' and 'hi2' respectively. However, if I provide a mock response only to the first acompletion request (only 'hi1'), it is being cached in the second response perfectly. 

Did not face this issue for the second test.